cmd/eval-dev-quality/cmd/evaluate.go

```diff
@@ -435,14 +435,19 @@ func (command *Evaluate) evaluateLocal(evaluationContext *evaluate.Context) (err
 
 // evaluateDocker executes the evaluation for each model inside a docker container.
 func (command *Evaluate) evaluateDocker(ctx *evaluate.Context) (err error) {
+
+	availableFlags := util.Flags(command)
 	ignoredFlags := []string{
 		"model",
 		"parallel",
 		"result-path",
 		"runtime",
 	}
+
+	// Filter all the args to only contain flags which can be used.
+	args := util.FilterArgs(os.Args[2:], availableFlags, false)
```

Comment:
Do we really need to do that? I though that the parsing errors anyways if we have additional flags, no?

Reply:
My problem during the docker testing was that the flags from the "testing" command where passend on to the docker runtime which caused errors. That is because `os.Args` has everything in it but the flags parser we are using does not store the actually parsed flags anywhere except as value on the struct itself.